src/stories/styleguide.mdx

```diff
@@ -0,0 +1,70 @@
+# Welcome to the GreenwoodJS style guide
+
+### The GreenwoodJS website styleguide breaks down into the following sections:
+
+1. Colors
+2. Typography
+3. Spacing
+4. Components
+
+
+
+## Colors
+The following theme colors available are:
+
+`--color-primary`
+
+`--color-secondary`
+
+`--color-tertiary`
+
+>> The --color-secondary is used for containers, while the --color-tetiary is used inside --color-secondary containers
+
+
+`--color-accent`
+
+`--color-gray`
+
+`--color-gray-inner`
+
+>> The --color-gray is used for containers, while the --color-gray-inner is used inside --color-gray containers
+
+
+`--color-header`
+
+`--color-text`
+
+`--color-white`
+
+
+
+## Typography
+The whole website uses Vercel's **Geist** font in the following sizes:
+
+`--hero-big`
```

Comment:
For the design system tokens, I don't think the goal is to try and capture every token used throughout all pages like these `--hero` specific ones, but rather just the ones all other tokens should build / compose off of.

I think just the main colors, fonts, sizes, etc should be good to start out with.  We don't need to predict every token right now, and this can totally evolve and grow as we build out the rest of the pages.  🏃‍♂️